src/content/apply/brand-designer.md

```diff
@@ -9,14 +9,14 @@ qualities:
   - Communicates well
   - Is kind
   - Can commit to 10 hours a week
-formLink: "https://sandboxf24.fillout.com/branddesignerapplication"
+formLink: "https://sandboxsp25.fillout.com/branddesignerapplication"
 mailLink: "https://gmail.us3.list-manage.com/subscribe?u=3b3ae33f54203ab7a839ae529&id=c2570dd048"
```

Comment:
note for future I think we hit the cap on newsletter signups, this is outdated and will need to be updated at some point

Reply:
website re-write :prayge: